house/apps.py

```diff
@@ -1,6 +1,5 @@
 from django.apps import AppConfig
 
 
-class houseConfig(AppConfig):
-    default_auto_field = 'django.db.models.BigAutoField'
+class HouseConfig(AppConfig):
```

Comment:
I think we need this line: 
    default_auto_field = 'django.db.models.BigAutoField'
Are you sure we can remove it?

Reply:
I saw some examples without it, Im also not sure what is the purpose of this line